build.gradle

```diff
@@ -5,14 +5,15 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.android.tools.build:gradle:3.0.+'
```

Comment:
This should be a specific version. 

Aside from that, 3.0.0 is not yet released as stable. Please use the latest stable version instead.

Reply:
Sorry, will do this later!